### DISCUSSION TITLE
support is it flutter ？

### DISCUSSION THREAD

**Discussion by dbl520** [community/community#19339] [Copilot Conversations]
support is it flutter ？

**Comment by melvincwng** [comment]
@dbl520, if I'm not mistaken, Github Copilot should support dart language/flutter framework.

Reference: [Github Copilot Flutter Youtube Tutorial](https://youtu.be/m-qaJfdVFSM)

Hope this clarifies.

**Reply by melvincwng** [reply]
@dbl520, if this reply has helped you, please 'Mark as answer' so that others can learn from your question as well.

Thank you and have a nice day 😊
